pyhoma/client.py

```diff
@@ -84,6 +88,7 @@ def __init__(
         self.password = password
         self.server = server
 
+        self.setup: Setup | None = None
```

Comment:
FYI the | operator is only available since 3.10, and pyhoma require at least 3.9.

Reply:
Ah, we use it in Home Assistant as well. Should I change it back, or just upgrade to 3.10?

However, doesn't `future annotations` solve this issue?

Reply:
Honnestly, I don’t know how to handle this properly.

Reply:
By the way, this works already, look at the Python version of our test suite.

'''
from __future__ import annotations
'''

Does the trick